contrib/initramfs/zfsunlock

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -u
```

Comment:
`set -e` has disappeared.

Reply:
put || true next to load-key, how non-null exit code does not cause script to exit, so put -e back